From 3da5b992d98bee627dce1140ab7b24cbaee3dbfb Mon Sep 17 00:00:00 2001
From: Luwei Zhou <b45643@freescale.com>
Date: Wed, 21 Aug 2013 13:14:24 +0800
Subject: [PATCH 0092/1074] ENGR00275999 Makefile:Support mma8451 sensor on
 i.MX6Q/DL SD/AUTO

Add mma8451 in Makefile.

Signed-off-by: Luwei Zhou <b45643@freescale.com>
[Original patch taken from git://git.freescale.com/imx/linux-2.6-imx.git]
Signed-off-by: Biyao Zhai <biyao.zhai@windriver.com>
---
 drivers/hwmon/Makefile |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/drivers/hwmon/Makefile b/drivers/hwmon/Makefile
index ec7cde0..1d312fe 100644
--- a/drivers/hwmon/Makefile
+++ b/drivers/hwmon/Makefile
@@ -142,6 +142,7 @@ obj-$(CONFIG_SENSORS_W83L785TS)	+= w83l785ts.o
 obj-$(CONFIG_SENSORS_W83L786NG)	+= w83l786ng.o
 obj-$(CONFIG_SENSORS_WM831X)	+= wm831x-hwmon.o
 obj-$(CONFIG_SENSORS_WM8350)	+= wm8350-hwmon.o
+obj-$(CONFIG_MXC_MMA8451)       += mxc_mma8451.o
 
 obj-$(CONFIG_PMBUS)		+= pmbus/
 
-- 
1.7.5.4

